pkg/validations/utils.go

```diff
@@ -10,6 +10,7 @@ type ValidationResult struct {
 	Name        string
 	Err         error
 	Remediation string
+	FeatureFlag bool
```

Comment:
Can we call this "Silent"? It's a bit more flexible and I think represents a bit better what it does
```suggestion
	Silent bool
```

Reply:
let's deal with this in another pr please, this is getting merged

Reply:
let's put this in another PR, this is going to be merged momentarily so we can kick off the staging release